### ISSUE TITLE
Outdated instructions ?

### ISSUE THREAD

**Issue by MICU-ADMIN** [alexandercerutti/passkit-generator#185]
Hi the instructions on creating the certificates may be outdated for example when i create my certificate and try to export it from keychain it doesnt allow to export as p12

**Comment by alexandercerutti** [comment]
Hi @MICU-ADMIN, thanks for the issue and thanks for using passkit-generator!

Which version of MacOS are you working on?
Instructions should not be outdated.

Also, may I ask you to provide a few screenshots?

Thank you!

**Comment by holman** [comment]
fwiw, it does seem to be out-of-date; Keychain on Sequoia only imports a single certificate and doesn't let you export to .p12.

**Comment by alexandercerutti** [comment]
@holman hey, thanks for the heads up! I'll be checking this asap

**Comment by holman** [comment]
Ah, sorry- was deep in C E R T I F I C A T E S and forgot to update this thread afterwards.

I think it still generally is correct (although new UI changes in Keychain Access has adjusted a few things, iirc). My main problem was that I was using a different name/email address in the CSR. You'd think that would be relatively okay, but Keychain doesn't count that as "your" certificate, so it doesn't group it in with your pub/priv keys so you can't export to .p12.

All in all, maddening (thanks Apple!). But if anything on this end, might want to call out "use your Keychain user info" instead of your company's information (for example), otherwise it breaks the flow with no real rationale as to *why*.

**Comment by alexandercerutti** [comment]
@holman wow that sounds _really_ funny ahaha I'm curious now to check this

**Comment by alexandercerutti** [comment]
@holman I'm on Sequoia and I tried to use a different email, through iCloud hide-my-email, to generate a CSR.

I still was able to find the imported certificate under "Certificates".

BTW, apparently the email addressed Keychain access is able to find, are those in your own contact in the Contacts app, because I remember I set up an amount of email addresses and I found them all in the suggestions.

So, I tried to use hide-my-email to generate a new CSR. Uploaded it to Apple Developer, downloaded the certificate, imported and with the right click, I was able to export it as P12 (I get suggested the format).

Honestly I don't know why you couldn't export it as P12...

**Comment by holman** [comment]
I was able to find it under "Certificates", but not "My Certificates"- I guess the difference there is (theoretically) why it can't be exported? Alternatively maybe it's a problem with the name instead of the email field. Both of which makes no sense to me as to why Apple would restrict one or the other or both of them! It's doubly weird because I was trying to use the *email* I have registered at Apple Developer, but that's not my local machine user so you'd think I'd be able to use that and ugh, what a mess. So strange.
